vofs/vofs/tests/test_vofs.py

```diff
@@ -1,22 +1,21 @@
-from __future__ import print_function
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
 from builtins import str
 from builtins import object
 import copy
-import requests
 import os
 import shutil
 import stat
 import ctypes
 from fuse import FUSE, FuseOSError, fuse_operations
-from contextlib import nested
 from errno import EIO, EAGAIN, EPERM, ENOENT
 import unittest2 as unittest
 from mock import Mock, MagicMock, patch, ANY
-from .. import vofs
+from vofs import vofs
```

Comment:
tests are not part of the vofs package. The .. syntax implies that either the test is a script (has a main) or is a subpackage in the vofs module.